crates/matrix-sdk-ui/src/timeline/event_item/mod.rs

```diff
@@ -382,7 +382,10 @@ impl EventTimelineItem {
                     Some(info.verification_state.to_shield_state_lax())
                 }
             }
-            None => Some(ShieldState::Grey { message: SENT_IN_CLEAR }),
+            None => Some(ShieldState::Grey {
```

Comment:
Sent in clear in an encrypted room is high level warning, should be red

Reply:
Ah my bad, updated 👍